gnosis/safe-react
# chore: Only show Whats new if there is a beamer app id

## What it solves
Relates to #3413 

Resolves https://github.com/gnosis/safe-react/pull/3515#issuecomment-1042845560 

## How this PR fixes it
If no Beamer app id exists, the "Whats New" Button stays hidden and Beamer is not included.

## How to test it
1. Open the Safe app on production without a beamer app id in `.env`
2. Observe that there is no "Whats new" Button in the sidebar
3. Accept all cookies
4. Observe that the console shows a Beamer app id missing warning
